site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "High Availability Agones"
+date: 2023-02-08T15:02:23Z
+weight: 20
+description: >
+  Learn how to configure your Agones services for high availability.
+---
+
+## High Availability for Agones Controller
+
+When `SplitControllerAndExtensions` is enabled, the `agones-controller` responsibility is split up into `agones-controller`, which enacts the Agones control loop, and `agones-extensions`, which acts as a service endpoint for webhooks and the allocation extension API. Splitting these responsibilities allows the `agones-extensions` pod to be horizontally scaled, making the Agones control plane highly available.
+
+{{< alpha title="`split controller and extensions`" gate="SplitControllerAndExtensions" >}}
+
+{{< alert title="GKE Autopilot Clusters" color="warning" >}}
+`SplitControllerAndExtensions` must be enabled for GKE Autopilot
+{{< /alert >}}
+
+## Extension Pod Configrations 
+
+It is important note is that `helm --set` sets the variable after the yaml inheritance and after helm resolves the chart values. Changing `controller` value through the use of `--set` will not change the values that `extensions` inherited from `controller`. The same applies for when `--values` or `-f` is used. 
```

Comment:
Hmm, before when I was suggesting reworking this, I meant the whole section. I don't think you need to even mention the inheritance (add it to the values.yaml as a comment), here say something like (not verbatim, maybe massage this):

---

The `agones-extensions` binary has similar `helm` configuration to `agones-controller`, see [reference here](insert link to helm variables). If you previously overrode `agones.controller` settings, you may need to override the same `agones.extensions` setting.

---

Also, we need to fix up the configuration table at https://e4313d7-dot-preview-dot-agones-images.appspot.com/site/docs/installation/install-agones/helm/#configuration.

Reply:
What the suggestion for `agones.extensions.*` settings? Should we just say somewhere that extensions has the same helm fields as `agones.controller` or should we have those fields listed out in the config doc as well?

Reply:
Oh, I missed that you asked this. I think we need to duplicate the documentation.